tests/sentry/tasks/integrations/github/test_open_pr_comment.py

```diff
@@ -328,7 +328,7 @@ def test_get_projects_and_filenames_from_source_file(self):
         ]
 
         project_list, sentry_filenames = get_projects_and_filenames_from_source_file(
-            self.organization.id, filename
+            self.organization.id, self.gh_repo.id, filename
```

Comment:
Can we have a separate more explicit test for filtering by repo id?

Reply:
sure

Reply:
done!